content/docs/user-guide/components/reference/atom/hdr-color-grading.md

```diff
@@ -0,0 +1,68 @@
+---
+title: HDR Color Grading Component
+linktitle: HDR Color Grading
+description: 'Open 3D Engine (O3DE) HDR Color Grading component reference.'
+toc: true
+---
+
+The **HDR Color Grading** component is a post-process effect that enables users to color grades scenes matching a desired look and feel. 
+This component uses a shader behind the scenes to perform the color grading logic on each frame's pixel. To reduce compute time, a Look-up (LUT) texture can be generated from the current color grading settings by clicking `Generate LUT` on the component. 
+![HDR Color Grading Generate LUT](/images/user-guide/components/reference/atom/post-processing-modifiers/hdr-color-grading-generate-lut-ui.png)
+
+The generated LUT texture can then be activated by clicking on the `Activate LUT` button on the component. 
+![HDR Color Grading Activate LUT](/images/user-guide/components/reference/atom/post-processing-modifiers/hdr-color-grading-activate-lut-ui.png)
+
+Upon activation, a **Look Modification** component is added to the current entity and subsequently the **HDR Color Grading** component is disabled. 
+
+## Provider ##
+
+[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
+
+## Dependencies
+
+[PostFX Layer component](/docs/user-guide/components/reference/atom/postfx-layer/)
+
+## Base properties
```

Comment:
```suggestion
## Properties
```
Can you separate the properties by property group in their own subsection? "Base properties" refers to properties that don't belong to a property group. Since all the properties here belong to a property group, then you can just call this section ## Properties and create ### subsections for the other property groups. Instead of one big table, you'll have multiple tables like this: 
> ## Properties
> ### LUT Generation properties
> ![image](https://user-images.githubusercontent.com/75444793/163280950-54a90657-2c3d-4a87-ac90-3cadf41734d4.png)
> `<insert properties table>`
> ### Color Adjustment properties
> ![image](https://user-images.githubusercontent.com/75444793/163280978-3cae2ac5-48a4-4b8b-9669-eedb44cd2231.png)
> `<insert properties table>`
>